NitroxClient/GameLogic/PDAManagerEntry.cs

```diff
@@ -12,9 +12,12 @@ public class PDAManagerEntry
         private readonly IPacketSender packetSender;
         public static Dictionary<NitroxTechType, PDAProgressEntry> CachedEntries { get; set; }
 
+        public bool CrashedUpdate;
```

Comment:
Would name it something like "AuroraExplosionTriggered" to clearly describe it's use case.